### PR TITLE
fix performance issue in updateSourceSpace of OpenAL implementation

### PR DESCRIPTION
**Pull request by nathhB**
https://github.com/emscripten-core/emscripten/issues/15847

### CODE REVIEW

File: src/library_openal.js [hunk 1]
```diff
@@ -609,19 +609,19 @@ var LibraryOpenAL = {
       }
 
       if (panner.positionX) {
-        panner.positionX.value = posX;
-        panner.positionY.value = posY;
-        panner.positionZ.value = posZ;
+        if (posX != panner.positionX.value) panner.positionX.value = posX;
```
Comment (sbc100) [inline src/library_openal.js:612]: Perhaps a quick comment to say why we do this?

```
// Assigning to panner.positionX/Y/Z unnecessarily seems to cause performance issues
// See https://github.com/emscripten-core/emscripten/issues/15847
```

**Comment by kripken** [comment]
(Also please merge in latest `main`, that should fix those test errors)

**Comment by nathhB** [comment]
> 

Did that, there is one test still failing in the "test-mac" node it seems.

**Comment by sbc100** [comment]
I re-started the mac tests.

**Pull request merged**